### ISSUE TITLE
Variables starting with g are highlighted.

### ISSUE THREAD

**Issue by palm86** [modelica-tools/atom-language-modelica#16]

**Comment by thorade** [comment]
Can you give an example, ideally as a complete .mo file uploaded to http://gist.github.com/ please?
Please also note that PR #13 might replace the complete grammar,
so fixing this issue will be delayed until PR #13 is merged (or rejected).
But we are still interested in your gist file for testing...

**Comment by dietmarw** [comment]
Here is a simple example:

``` modelica
record gTest
  parameter Real foo;
  parameter Real gfoo;
  parameter Real bar;
  parameter Integer gbar;

end gTest;
```

gives in atom.io (lightshow for some reason does not show this error ... )
![screenshot from 2015-10-15 19 20 09](https://cloud.githubusercontent.com/assets/9332/10521761/d7e45518-7371-11e5-980c-cac476e8e342.png)

**Comment by dietmarw** [comment]
Forgot to mention that this works fine in modelica-mode of emacs :stuck_out_tongue:

**Comment by thorade** [comment]
Just tested it, this would also be closed by PR #13

**Comment by thorade** [comment]
Fixed by PR #13
If not, feel free to reopen.
